changelog/changes/2024-03-12-remove-mlnx-switch-asics-drivers.md

```diff
@@ -0,0 +1 @@
+- Removed Linux drivers for Mellanox Technologies Switch ASICs family and Spectrum/Spectrum-2/Spectrum-3/Spectrum-4 Ethernet Switch ASICs.
```

Comment:
```suggestion
- Removed Linux drivers for Mellanox Technologies Switch ASICs family and Spectrum/Spectrum-2/Spectrum-3/Spectrum-4 Ethernet Switch ASICs to reduce the initrd size ([scripts#1734](https://github.com/flatcar/scripts/pull/1734))
```

Reply:
@pothos I added also the size freed and the issue number of the overall effort.

Reply:
Ok, it's a bit much detail for users but why not^^